setup.py

```diff
@@ -8,7 +8,7 @@
 install_requires = [
     'docutils',
     'python-dateutil',
-    'pyparsing',
+    'pyparsing>=2',
```

Comment:
As you pointed out the new function was added in version 1.5.7. Why should the minimum version be 2 then - and not 1.5.7?

Reply:
Because 2 has been around plenty long (since 2012!) and I doubt we *need* compatibility with 3 major release versions, so it seemed like a reasonable place to put it. I have "Allow edits from maintainers" enabled, so feel free to amend.

Reply:
Well, I guess I have to apply the changes myself then if that is too much to ask from you.